bundles/company-users-bulk-rest-api/src/FondOfOryx/Shared/CompanyUsersBulkRestApi/Transfer/company_users_bulk_rest_api.transfer.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<transfers xmlns="spryker:transfer-01"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="spryker:transfer-01 https://static.spryker.com/transfer-01.xsd">
+    <transfer name="RestCompanyUsersBulkRequestAttributes">
+        <property name="assign" type="RestCompanyUsersBulkItem[]" singular="item"/>
+        <property name="unassign" type="RestCompanyUsersBulkItem[]" singular="item"/>
+    </transfer>
+
+    <transfer name="RestCompanyUsersBulkRequest">
+        <property name="attributes" type="RestCompanyUsersBulkRequestAttributes"/>
+        <property name="originatorReference" type="string"/>
+    </transfer>
+
+    <transfer name="RestCompanyUsersBulkResponse">
+        <property name="isSuccessful" type="bool"/>
+        <property name="error" type="string"/>
+        <property name="code" type="int"/>
```

Comment:
Maybe you can use an error message transfer.

Reply:
Note that the message must be set as a glossary key. Later you need the glossary storage client for the correct translation.